class-job.php

```diff
@@ -51,6 +51,8 @@ public function save() {
 			$data['interval'] = $this->interval;
 		}
 
+		wp_cache_delete( "jobs:{$this->site}", 'cavalcade-jobs' );
```

Comment:
Oh right, missed the "non-persistent" part of your comment.

Turns out I was overthinking this a bit; I forgot that the plugin and the runner both have _separate_ Job classes, I was thinking this code was loaded in the runner, and hence wouldn't have the cache functions declared.

Using the cache functions is hence fine, although as @joehoyle noted, the site ID is already included in the cache key. Since this code isn't loaded in the runner, it doesn't matter whether the runner sets the globals or not :)